### ISSUE TITLE
Mac support?

### ISSUE THREAD

**Issue by Hxrdty** [TobyAdd/GDH#242]
if possible could this genode also be mac support aswell

**Comment by Rustring** [comment]
No Mac support I think

**Comment by Rustring** [comment]
Yeah the menu doesn't work for MacOS, based on how it's made, use Prism Menu or QOLMod instead.

![Screenshot_20240709_142908](https://github.com/TobyAdd/GDH/assets/155897302/44bc65dd-43af-4ee5-81d4-6097b79e6987)

**Comment by Superpro123411** [comment]
> Yeah the menu doesn't work for MacOS, based on how it's made, use Prism Menu or QOLMod instead.
> 
> ![Screenshot_20240709_142908](https://private-user-images.githubusercontent.com/155897302/346903026-44bc65dd-43af-4ee5-81d4-6097b79e6987.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE0ODI5MjIsIm5iZiI6MTcyMTQ4MjYyMiwicGF0aCI6Ii8xNTU4OTczMDIvMzQ2OTAzMDI2LTQ0YmM2NWRkLTQzYWYtNGVlNS04MWQ0LTYwOTdiNzllNjk4Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyMFQxMzM3MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xOTI2ZjZjYTBhNTA0MzY2NjM0MzdkNTkzZjQ5MzhjYjA1NGZmYTAzNjVhMWQyNDJmNzhhZjg5MDFhMjliMzEwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.hYKjeeoNuQjkxpzjw71PQ1fjdSeVVeSybeMu4od5lZc)

or Eclipse (though its in Beta)

**Comment by Rustring** [comment]
> > Yeah the menu doesn't work for MacOS, based on how it's made, use Prism Menu or QOLMod instead.
> > ![Screenshot_20240709_142908](https://private-user-images.githubusercontent.com/155897302/346903026-44bc65dd-43af-4ee5-81d4-6097b79e6987.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE0ODI5MjIsIm5iZiI6MTcyMTQ4MjYyMiwicGF0aCI6Ii8xNTU4OTczMDIvMzQ2OTAzMDI2LTQ0YmM2NWRkLTQzYWYtNGVlNS04MWQ0LTYwOTdiNzllNjk4Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyMFQxMzM3MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xOTI2ZjZjYTBhNTA0MzY2NjM0MzdkNTkzZjQ5MzhjYjA1NGZmYTAzNjVhMWQyNDJmNzhhZjg5MDFhMjliMzEwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.hYKjeeoNuQjkxpzjw71PQ1fjdSeVVeSybeMu4od5lZc)
> 
> or Eclipse (though its in Beta)

No you're wrong

![Screenshot_20240721_145326](https://github.com/user-attachments/assets/7b1188c9-09e0-41ed-9e99-c52d3656e487)

**Comment by Superpro123411** [comment]
i said You could use Eclipse for Macos

**Comment by mangodev01** [comment]
wait there's no mac support?
It's dear imgui, you could add that, it's not like there's a limitation or smth
